spec/acceptance/build_stubbed_spec.rb

```diff
@@ -167,7 +167,7 @@
   it "allows overriding created_at for objects with created_at" do
     created_at = 3.days.ago
     stubbed = build_stubbed(:thing_with_timestamp, created_at: created_at)
-    expect(stubbed.created_at).to eq created_at
+    expect(stubbed.created_at).to be_within(1.second).of created_at
```

Comment:
Has something changed in Rails that was causing this test to fail? Are we getting back a different precision after the database round trip, perhaps?

Reply:
@composerinteralia Yes, this is a precision issue.  We get a difference in the milliseconds as I recall from running the specs.  During the round trip it appears to be rounded to 0.